### PR TITLE
Fix: Manually dispatch scroll event

### PR DESCRIPTION
**Pull request by ConradJChan**
Firefox and IE11 didn't fire the scroll event when setting `scrollTop`, so this fix is to manually dispatch the scroll event in `VirtualScroller` so that the items at that scroll point will get rendered

### CODE REVIEW

**Comment by boxcla** [comment]
Verified that @ConradJChan has signed the CLA. Thanks for the pull request!

File: src/lib/VirtualScroller.js [hunk 1]
```diff
@@ -421,6 +421,10 @@ class VirtualScroller {
             // will get rendered.
             const topPosition = (this.itemHeight + this.margin) * rowIndex;
             this.scrollingEl.scrollTop = topPosition;
+            // Some browsers don't fire the scroll event when setting scrollTop
+            // (IE11 & Firefox) so we need to manually dispatch the event
+            // in order to trigger `onScrollHandler` to render the items
+            this.scrollingEl.dispatchEvent(new Event('scroll'));
```
Comment (jeremypress) [inline src/lib/VirtualScroller.js]: Any issues with the event firing twice on browsers that automatically trigger it?

Comment (ConradJChan) [reply src/lib/VirtualScroller.js]: I didn't see it firing twice, but I'll double check

Comment (ConradJChan) [reply src/lib/VirtualScroller.js]: Yeah you're right it fires twice, but the `scrollTop` is the same so the logic to render items won't get triggered the second invocation

**Pull request merged**